src/Connection.php

```diff
@@ -31,6 +31,22 @@ class Connection
      */
     public $driverType;
 
+    /**
```

Comment:
commented here: https://github.com/atk4/dsql/pull/196#discussion_r443658293

`driverType` can make sense if connection is wrapped, but I think we does not even support this scenario now (and we should not, we should support transparent layering)

for this reaon, I think we can deprecate this property in advance to native `instanceof` operator

Reply:
@georgehristov but the driverType should be removed, it does not work with nested drivers... (to check if the underlaying driver is of some type)

As the check was not required until now, I would not introduce any helper method for it now until really required.